notebooks/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py

```diff
@@ -63,7 +63,8 @@
 # EPY: END markdown
 
 # EPY: START code
-codebook = pd.read_csv('MERFISH/codebook.csv', dtype={'barcode': object})
+# TODO ambrosejcarr: update this to point at the s3 bucket
```

Comment:
remove this TODO?

Reply:
Will fix so that it points at the bucket. 👍